### DISCUSSION TITLE
Charting with keybinds

### DISCUSSION THREAD

**Discussion by THEkatinamicrowave** [CodenameCrew/CodenameEngine#428] [Ideas]
@NexIsDumb hear me out

Chart editor.
_playback_play().
typing using the keybinds to add notes to the chart.

this would speed up charting INSANELY QUICK however the catch is that you have to be good at charting on the fly.
Also the notes during keybind-charting would obviously be locked to a custom grid that you can make.
If the note sustain is noticeably short, it will decide you didn't mean to hold the note and it won't do anything

neat idea that would speed up charting

**Comment by r6915ee** [comment]
This might be something that can be transferred to the issue tracker under the `enhancement` label, though I still do like your idea.

**Reply by THEkatinamicrowave** [reply]
yeah I think it would be cool also because I'm porting psych charts to codename and A) I'm too productive to just drag and drop the psych charts into codename but also I'm B) too lazy to continuously swap between Psych and Codename windows every single measure

I'm tryna figure out Charter.hx and, using that information, edit the file to the point where it can be made into a fork
